.gitignore

```diff
@@ -2,4 +2,5 @@
 /tests/results
 /tests/resources/app/coldbox
 /node_modules
-/modules
\ No newline at end of file
+/modules
+.vscode/settings.json
```

Comment:
I'd suggest adding this to your own global gitignore file so you don't have to add it to every repo you contribute to. https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Good suggestion.  I'm pretty new to all of this and I appreciate your patience with me.